ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -1412,33 +1412,34 @@ export function nodePortsFactory(ports?: Partial<NodePort>[]): NodePort[] {
 export function genericNodeFactory(
   {
     id,
-    name,
+    label,
     nodeTrigger,
     nodePorts,
     nodeAttributes,
     nodeOutputs,
     adornments,
   }: {
     id?: string;
-    name: string;
+    label: string;
     nodeTrigger?: NodeTrigger;
     nodePorts?: NodePort[];
     nodeAttributes?: NodeAttribute[];
     nodeOutputs?: NodeOutput[];
     adornments?: AdornmentNode[];
   } = {
-    name: "MyCustomNode",
+    label: "MyCustomNode",
   }
 ): GenericNode {
   const nodeData: GenericNode = {
     id: id ?? uuidv4(),
+    label: label,
     type: WorkflowNodeType.GENERIC,
     base: {
       module: ["vellum", "workflows", "nodes", "bases", "base"],
       name: "BaseNode",
     },
     definition: {
-      name,
+      name: label,
```

Comment:
[nit] we likely want to throw this into whatever class name normalization fcn we have